df_chat/tests/test_chat.py

```diff
@@ -247,6 +249,88 @@ async def test_chat_multiple_rooms(self):
         await communicator2.disconnect()
         await communicator3.disconnect()
 
+    def test_create_message(self):
+        User = get_user_model()
+        user = User.objects.create_superuser(username="test_username", password="test_password")
+        self.client.login(username='test_username', password='test_password')
+        room_create_response = self.client.post(
+            "/api/v1/chat/rooms/",
+            data={"title": "test_room", "users": [user.id]},
+        )
+        self.assertEqual(room_create_response.status_code, http.HTTPStatus.CREATED)
+        room_id = room_create_response.json()["id"]
+        message_create_response = self.client.post(
+            f"/api/v1/chat/rooms/{room_id}/messages/",
+            data={"body": "Test message 1234."},
+        )
+        self.assertEqual(message_create_response.status_code, http.HTTPStatus.CREATED)
+    
+    def test_list_message(self):
```

Comment:
Could you please add a docstring that explains the purpose of the test you are running here?

Reply:
Done!